eleventy.config.ts

```diff
@@ -108,6 +142,15 @@ if (process.env.WCAG_MODE === "editors") {
   baseUrls.understanding = `https://www.w3.org/WAI/WCAG${version}/Understanding/`;
 }
 
+/** Applies any overridden SC IDs to incoming Understanding fileSlugs */
+function resolveUnderstandingFileSlug(fileSlug: string) {
+  if (fileSlug in scSlugOverrides) {
+    assertIsWcagVersion(version);
+    return scSlugOverrides[fileSlug](version);
+  }
+  return fileSlug;
```

Comment:
If someone somehow tries to visit the non-overriden slug, this is still going to resolve it; if I try that for `target-size-enhanced` in your test branch (ie, https://kfranqueiro.github.io/wcag/understanding/target-size-enhanced ), it resolves, but the resulting page is mistitled using the 2.2 title instead of the 2.1 title. Perhaps better to prevent it from resolving?

Reply:
Thanks for taking a detailed look at this. The `-enhanced` page you are seeing is a remnant from what was already on the `gh-pages` branch that this published over; when running a clean build with `WCAG_VERSION=21`, only `target-size.html` is produced.

```
$ ls _site/understanding/target-size*
_site/understanding/target-size.html
```

The way that `scSlugOverrides` works is that each input file's basename (i.e. `target-size-enhanced`) gets looked up, and if a function for that name is found, the output of the function is used as the output basename instead. This results in `target-size-enhanced` being mapped (or "moved") to `target-size` specifically for WCAG < 2.2; there is no "fallback".